PhysicsTools/Heppy/src/TriggerBitChecker.cc

```diff
@@ -45,33 +42,14 @@ namespace heppy {
     }
     bool outcome = true;
     for (std::vector<unsigned int>::const_iterator it = indices_.begin(), ed = indices_.end(); it != ed; ++it) {
-      if (result.getPrescaleForIndex(*it) != 1) {
+      if (result.getPrescaleForIndex<double>(*it) != 1) {
```

Comment:
Is this ok with rounding effects?

Reply:
See https://github.com/cms-sw/cmssw/pull/39220#discussion_r975393631.